packages/node/src/types.ts

```diff
@@ -285,8 +286,9 @@ export interface ChainProvider {
 }
 
 export interface ChainConfig {
-  readonly id: number;
+  readonly adminAddress: string;
```

Comment:
This is not needed if the provider record has already been created. Should we make this optional? We would be increasing the complexity of what a valid `config.json` is, but having the user enter a placeholder is also bad.

Reply:
Assuming we don't change the structure of `config.json`, I prefer to just leave it as required. I think having certain fields required and then later optional will just end up more confusing. 

Plus it means more checks/error catching in the node if the field exists before accessing it.

What do you think?

Reply:
The only problem is that the user will change the `adminAddress` in `config.json` thinking that this will do anything after the provider is created, but having this field optional doesn't solve this problem.

Renaming this here and in `config.json` should help though, something like `adminAddressForCreatingProviderRecord`.